hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestSparkConsistentBucketClustering.java

```diff
@@ -110,7 +115,7 @@ public void setup(int maxFileSize, Map<String, String> options) throws IOExcepti
         .withStorageConfig(HoodieStorageConfig.newBuilder().parquetMaxFileSize(maxFileSize).build())
         .withClusteringConfig(HoodieClusteringConfig.newBuilder()
             .withClusteringPlanStrategyClass(SparkConsistentBucketClusteringPlanStrategy.class.getName())
-            .withClusteringExecutionStrategyClass(SparkConsistentBucketClusteringExecutionStrategy.class.getName())
+            .withClusteringExecutionStrategyClass(singleJob ? SINGLE_SPARK_JOB_CONSISTENT_HASHING_EXECUTION_STRATEGY : SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY)
```

Comment:
Is `SINGLE_SPARK_JOB_CONSISTENT_HASHING_EXECUTION_STRATEGY` always better than `SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY`, why we need two execution strategy.

Reply:
> Is `SINGLE_SPARK_JOB_CONSISTENT_HASHING_EXECUTION_STRATEGY` always better than `SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY`, why we need two execution strategy.

I'm not sure if any user is already using `SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY` , and if so, should we keep it for compatibility? If not, we can deprecated it.